configuration/parse.go

```diff
@@ -91,6 +105,32 @@ Options:
 	return config, nil
 }
 
+func loadConfigFromFile(configLocation string, config *Configuration) error {
+	b, err := ioutil.ReadFile(configLocation)
```

Comment:
Another thought (I'll add it to #111) is to refactor the `loadConfigFromFile()` and `loadIQConfigFromFile()` methods to take `io.Reader` - instead of a path `configLocation`. The tests could then easily pass in bytes representing the various file contents. This would also allow the test avoid pesky disk access.